python/lsst/ip/isr/isrStatistics.py

```diff
@@ -215,11 +254,21 @@ def run(self, inputExp, ptc=None, overscanResults=None, **kwargs):
         if self.config.doCopyCalibDistributionStatistics:
             calibDistributionResults = self.copyCalibDistributionStatistics(inputExp, **kwargs)
 
+        biasShiftResults = None
+        if self.config.doBiasShiftStatistics:
+            biasShiftResults = self.measureBiasShifts(inputExp, overscanResults)
+
+        ampCorrelationResults = None
+        if self.config.doAmplifierCorrelationStatistics:
+            ampCorrelationResults = self.measureAmpCorrelations(inputExp, overscanResults)
+
         return pipeBase.Struct(
-            results={"CTI": ctiResults,
-                     "BANDING": bandingResults,
-                     "PROJECTION": projectionResults,
+            results={'CTI': ctiResults,
```

Comment:
I looks like one of these is still double quotation marks?

Reply:
I've fixed all instances of single quotes in this file.